src/vdaf/prio3.rs

```diff
@@ -364,8 +361,8 @@ where
         Ok(out)
     }
 
-    #[cfg(test)]
-    pub(crate) fn test_vec_shard(
+    /// Shard measurement with constant randomness of repeated 0x01 bytes.
```

Comment:
Why is this change needed? (why does test_vec_shard need to be pub?)

Reply:
If I want to do tests similar to what we are doing internally for my external implementation it seems I need it. See here: https://github.com/simon-friedberger/priouser/blob/5ed81adeed2521ad43c3e09b438bd5e03fbac391/src/main.rs#L156
Is there a better way?

Reply:
Ah, it's really too bad that turning on `#[cfg(test)]` in your crate doesn't also enable it in dependencies you pull in. I think the best way forward here would be to do [what Tokio does](https://docs.rs/tokio/latest/tokio/#feature-flags) and [what we did in Janus](https://github.com/divviup/janus/blob/4716fe32d77b9f4dc55503df9e49534e214cf5e8/janus_core/Cargo.toml#L14) and gate things like this behind a `test-util` feature. We already have a `test-vector` feature that we use to conditionally compile the backward compatibility tests, so we could extend that to cover this and any other items you need to enable exclusively in your test code.

Reply:
Oh, Ok gotcha. Can you add ore context to the comment?
```suggestion
    /// Shard measurement with constant randomness of repeated bytres.
    ///
    /// This method is not secure. It is used for running test vectors for Prio3. 
```

Reply:
(Or taking @tgeoghegan's suggestion would also be sufficient as far as I'm concerned.)